nidm_results_display.m

```diff
@@ -25,7 +25,15 @@
         addpath(fullfile(fileparts(mfilename('fullpath')), 'lib'));
     end
     
+    graph = jsondoc.('x_graph');
+    % Deal with sub-graphs (bundle)
```

Comment:
The sub-graph is created by the prov library to represent the NIDM-Results bundle (this was not the case in the jsons you were creating from the turtle because turtle does not have an explicit way to represent bundles).